app/DynamicsAdapter/DynamicsAdapter.Web/Mapping/Converters.cs

```diff
@@ -212,5 +212,39 @@ public class IncaceratedConverter : IValueConverter<string, int?>
         }
     }
 
-    
+    public class RelatedPersonCategoryConverter : IValueConverter<string, int?>
+    {
+        public int? Convert(string source, ResolutionContext context)
+        {
+            if ("aunt/uncle".Contains(source.ToLower())) 
+                return PersonRelationType.AuntUncle.Value;
+            else
+                return
+                    source.ToLower() switch
+                    {
+                        "spouse" => PersonRelationType.Spouse.Value,
+                        "parent" => PersonRelationType.Parent.Value,
+                        "child" => PersonRelationType.Child.Value,
+                        "sibling" => PersonRelationType.Sibling.Value,
+                        "cousin" => PersonRelationType.Cousin.Value,
+                        "friend" => PersonRelationType.Friend.Value,
+                        _ => (int?)null
+                    };
+        }
+    }
+
+    public class RelatedPersonGenderConverter : IValueConverter<string, int?>
+    {
+        public int? Convert(string source, ResolutionContext context)
+        {
+            return
+                source.ToLower() switch
+                {
+                    "m" => GenderType.Male.Value,
+                    "f" => GenderType.Female.Value,
+                    "u" => GenderType.Other.Value,
+                    _ => (int?)null
+                };
+        }
+    }
 }
```

Comment:
can we make this generic gender converter 'PersonGenderConverter'

Reply:
sure. will do.